File: cpp/bc1046.cpp
#include <iostream>

using namespace std;

int main()
{
	int hIni, hFin, dur;
	
	cin >> hIni >> hFin;
	
	hFin < hIni || hFin == hIni? dur = 24 + (hFin - hIni) : dur = hFin - hIni;
	
	cout << "O JOGO DUROU " << dur << " HORA(S)" << endl;
}

File: python/bc1046.py
cVal = input().split(" ")

mIni = int(cVal[0])
mFin = int(cVal[1])

res = 24 + (mFin - mIni) if (mFin < mIni) or (mFin == mIni) else mFin - mIni

print(f"O JOGO DUROU {res} HORA(S)")